docs/data_products/sections/derived_data.tex

```diff
@@ -28,26 +28,10 @@ \subsubsection{Pre-processing EKG and GSR Signals}
 
 \subsubsection{Synchronization of EEG, EKG, GSR, and fNIRS Signals}
 
-After the EEG, EKG, GSR, and fNIRS signals are pre-processed to remove noise, the signals are resampled to a common sampling rate using the FFT-based resampling method \texttt{mne.filter.resample} available in the Python MNE library \cite{GramfortEtAl2013a}. To synchronize the signals, we generate a time series matching the common sampling rate of the resampled signals, with the first timestamp rounded to the nearest second. Then, the signals are interpolated to this generated time series via linear interpolation.
+After the EEG, EKG, GSR, and fNIRS signals are pre-processed to remove noise, the signals are upsampled to 2000Hz using the FFT-based resampling method \texttt{mne.filter.resample} available in the Python MNE library \cite{GramfortEtAl2013a}. 
 
-\subsection{Synchronizing Task Data with EEG and fNIRS Resampled Signals}
+For each experiment, we define a common clock with initial time starting 2 minutes before the beginning of the first task (rest state) and end time set to 2 minutes after the final task (typically Minecraft). We create equally spaced ticks in this clock at the frequency of 200Hz. Then, the signals are downsampled to this clock's scale via linear interpolation. 
```

Comment:
_We created equally spaced ticks in this clock at the frequency of 200Hz_

Why 200 Hz, I thought you upsampled to 2000 Hz and synchronize it there. Shouldn't this be 200 Hz?

Reply:
@eduongAZ  --- I believe @paulosoaresua simply moved the tables describing the older version of the derived data to the appendix. Do I understand that correctly @paulosoaresua ?

Reply:
Yes, I moved them to the appendix. I kept the ones pertinent to the mapping between table columns and montage in the body of the document.

Reply:
About the frequency, there are 2: I upsample to 2000Hz and interpolate to the main clock which has 200Hz. The latter is the frequency of the signals saved in the sync tables.

Reply:
Ok, so instead of synchronizing at 2000 Hz at a 2000 Hz clock and then downsample, you upsample at 2000 Hz and then synchronize at 200 Hz directly.

I don't see any obvious issue with this approach.